### PR TITLE
Fix risk condition on loading indicator when layout completes before vsync.

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: test/functional/test-custom-element.js [hunk 1]
```diff
@@ -1002,6 +1002,28 @@ describe('CustomElement Loading Indicator', () => {
       expect(toggle.firstCall.args[1]).to.equal(true);
     });
   });
+
+  it('should ignore loading on if layout completed before vsync', () => {
```
Comment (erwinmombay) [inline test/functional/test-custom-element.js]: only if?

Comment (dvoytenko) [inline test/functional/test-custom-element.js]: It's loading "on" :)

Comment (dvoytenko) [inline test/functional/test-custom-element.js]: Done

File: src/custom-element.js [hunk 2]
```diff
@@ -305,6 +305,9 @@ export function createAmpElementProto(win, name, implementationClass) {
     /** @private {boolean|undefined} */
     this.loadingDisabled_;
 
+    /** @private {boolean|undefined} */
+    this.loadingState_;
```
Comment (erwinmombay) [inline src/custom-element.js]: is it possible to to start with a boolean state? like `false` by default?

Comment (dvoytenko) [inline src/custom-element.js]: To be stripped out by a compiler.

**Comment by erwinmombay** [comment]
LGTM.

**Pull request merged**